samples/csharp_dotnetcore/40.timex-resolution/Program.cs

```diff
@@ -1,24 +1,37 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System;
+using System.Collections.Generic;
+
 namespace Microsoft.BotBuilderSamples
 {
     class Program
     {
         static void Main(string[] args)
         {
-            // Creating TIMEX expressions from natural language using the Recognizer package.
-            Ambiguity.DateAmbiguity();
-            Ambiguity.TimeAmbiguity();
-            Ambiguity.DateTimeAmbiguity();
-            Ranges.DateRange();
-            Ranges.TimeRange();
+            var samples = new List<Action>
+            {
+                // Creating TIMEX expressions from natural language using the Recognizer package.
+                Ambiguity.DateAmbiguity,
+                Ambiguity.TimeAmbiguity,
+                Ambiguity.DateTimeAmbiguity,
+                Ranges.DateRange,
+                Ranges.TimeRange,
+
+                // Manipulating TIMEX expressions in code using the TIMEX Datatype package.
+                Parsing.Examples,
+                LanguageGeneration.Examples,
+                Resolution.Examples,
+                Constraints.Examples
+            };
 
-            // Manipulating TIMEX expressions in code using the TIMEX Datatype package.
-            Parsing.Examples();
-            LanguageGeneration.Examples();
-            Resolution.Examples();
-            Constraints.Examples();
+            foreach (var sample in samples)
```

Comment:
I don't see how this is better or in any way more readable! Just replacing a sequential set of functions with a list of functions that is then iterated over. This is just taking a perfectly natural and simple mechanism of code and replacing it with an artificial handcrafted mechanism. So I don't think we need this part at all.